server/src/main/java/com/cloud/api/ApiResponseHelper.java

```diff
@@ -2187,13 +2188,75 @@ public NetworkResponse createNetworkResponse(ResponseView view, Network network)
                         CapabilityResponse capabilityResponse = new CapabilityResponse();
                         String capabilityValue = ser_cap_entries.getValue();
                         capabilityResponse.setName(capability.getName());
-                        capabilityResponse.setValue(capabilityValue);
+                        if (Service.Lb == service && capability.getName().equals(Capability.SupportedLBIsolation.getName())) {
+                            // capabilityResponse.setValue(networkOffering.getDedicatedLB() ? "dedicated" : "shared");
```

Comment:
@ravening the rest is ok for me, except this line.

Reply:
@weizhouapache fixed it.